angular/src/directives/control-value-accessors/numeric-value-accessor.ts

```diff
@@ -18,13 +18,13 @@ export class NumericValueAccessorDirective extends ValueAccessor {
     super(injector, el);
   }
 
-  @HostListener('ionChange', ['$event.target'])
-  _handleIonChange(el: any): void {
+  @HostListener('ionInput', ['$event.target'])
+  handleInputEvent(el: HTMLIonInputElement): void {
     this.handleChangeEvent(el, el.value);
   }
```

Comment:
Is the name `handleChangeEvent` the confusing piece here? 

That function is responsible for performing the `onChange` callback for Angular's control value accessor as well as syncing the Ionic "status" classes (`.ion-invalid`, etc.). With us no longer listening to `ionChange`, I can see where that name is problematic. I want to avoid renaming it to `handleInputEvent` or similar, since currently only `ion-input` is updated to the revised approach.

Perhaps a rename like this would work?

```ts
@HostListener('ionInput', ['$event.target'])
handleInputEvent(el: HTMLIonInputElement): void {
  this.commitValueChange(el, el.value);
}
```

I don't think we are using the base class that comes from [`@stencil/angular-output-targets`](https://github.com/ionic-team/stencil-ds-output-targets/blob/main/packages/angular-output-target/resources/control-value-accessors/value-accessor.ts) so we have full control to refactor as necessary here.

Reply:
That works for me 👍